README.md

```diff
@@ -29,7 +29,7 @@
 The friendly Operating System for IoT!
 
 RIOT is a real-time multi-threading operating system that supports a range of
-devices that are typically found in the Internet of Things (IoT): 
```

Comment:
Might want to remove this

Reply:
```README.md``` is exactly that from master. The only one difference was a whitespace at the end of the line which let the ```whitespacecheck``` of TravisCI fail. So, I thought to correct it.

Reply:
Oh ya weird, I can't find it in the current master and I feel silly for talking about a whitespace.